keyboards/underscore33/rev1/config.h

```diff
@@ -1,10 +1,25 @@
+/* Copyright 2020 tominabox1
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
 #pragma once
 
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID 0xFEED
-#define PRODUCT_ID 0x6A50
+#define VENDOR_ID 0x7431
+#define PRODUCT_ID 0x0002
```

Comment:
VIA support requires each supported board to have a unique Vendor/Product ID pairing. (It doesn't look at `DEVICE_VER` at all.) You'll need two JSON files for VIA, one for each repo.

```suggestion
#define PRODUCT_ID 0x3301  // _33 rev1
```